nautobot/extras/models/models.py

```diff
@@ -519,6 +519,8 @@ class FileProxy(BaseModel):
     )
     uploaded_at = models.DateTimeField(auto_now_add=True)
 
+    _natural_key = ["name", "uploaded_at"]
```

Comment:
Not actually a unique together constraint presently but "close enough" since we're unlikely to have two files uploaded with the same name at the exact same timestamp.